public/apps/configuration/panels/service-account-list.tsx

```diff
@@ -0,0 +1,220 @@
+      render: truncatedListView(tableItemsUIProps),
+    },
+    {
+      field: 'attributes',
+      name: 'Attributes',
+      render: dictView,
+      truncateText: true,
+    },
+  ];
+}
+
+export function ServiceAccountList(props: AppDependencies) {
+  const [userData, setUserData] = React.useState<InternalUsersListing[]>([]);
+  const [errorFlag, setErrorFlag] = React.useState(false);
+  const [selection, setSelection] = React.useState<InternalUsersListing[]>([]);
+  const [currentUsername, setCurrentUsername] = useState('');
+  const [loading, setLoading] = useState(false);
+  const [query, setQuery] = useState<Query | null>(null);
+
+  React.useEffect(() => {
+    const fetchData = async () => {
+      try {
+        setLoading(true);
+        const userDataPromise = getUserList(props.coreStart.http, ResourceType.serviceAccounts);
+        setCurrentUsername((await getAuthInfo(props.coreStart.http)).user_name);
+        setUserData(await userDataPromise);
+      } catch (e) {
+        console.log(e);
+        setErrorFlag(true);
+      } finally {
+        setLoading(false);
+      }
+    };
+
+    fetchData();
+  }, [props.coreStart.http]);
+
+  const actionsMenuItems = [
+    <EuiButtonEmpty
+      data-test-subj="edit"
+      key="edit"
+      onClick={() => {
+        window.location.href = buildHashUrl(ResourceType.users, Action.edit, selection[0].username);
+      }}
+      disabled={selection.length !== 1}
+    >
+      Edit
+    </EuiButtonEmpty>,
+    <EuiButtonEmpty
+      data-test-subj="duplicate"
+      key="duplicate"
+      onClick={() => {
+        window.location.href = buildHashUrl(
+          ResourceType.users,
+          Action.duplicate,
+          selection[0].username
+        );
+      }}
+      disabled={selection.length !== 1}
+    >
+      Duplicate
+    </EuiButtonEmpty>,
+    <EuiButtonEmpty
+      key="export"
+      disabled={selection.length !== 1}
+      href={
+        selection.length === 1
+          ? `${props.coreStart.http.basePath.serverBasePath}${API_ENDPOINT_SERVICEACCOUNTS}/${selection[0].username}`
+          : ''
+      }
+      target="_blank"
+    >
+      Export JSON
+    </EuiButtonEmpty>,
+  ];
+
+  const [actionsMenu, closeActionsMenu] = useContextMenuState('Actions', {}, actionsMenuItems);
+
+  return (
+    <>
+      <EuiPageHeader>
+        <EuiTitle size="l">
+          <h1>Service accounts</h1>
+        </EuiTitle>
+      </EuiPageHeader>
+      <EuiPageContent>
+        <EuiPageContentHeader>
+          <EuiPageContentHeaderSection>
+            <EuiTitle size="s">
+              <h3>
+                Service accounts
+                <span className="panel-header-count">
+                  {' '}
+                  ({Query.execute(query || '', userData).length})
+                </span>
+              </h3>
+            </EuiTitle>
+            <EuiText size="s" color="subdued">
+              Here you have a list of special accounts that represent services like extensions,
```

Comment:
@samuelcostae @scrawfor99 @DarshitChanpura Are service accounts limited to the reserved (system) indices that the extension is associated with or will they be able to do more? I'd like to make sure the language here reflects what these accounts would be/are capable of.

Reply:
They should be limited to the indices reserved by the associated service (e.g an extension)

Reply:
@cwperks  Only thing I can think of that could be added is "Differently from normal accounts, Service Accounts are able to access the system_indices associated assigned to their role"

Reply:
This is fine as is.